android-sdk/templates/android-sdk-persistent-template.j2

```diff
@@ -132,7 +132,7 @@
                   }
                 ],
                 "terminationMessagePath": "/dev/termination-log",
-                "imagePullPolicy": "IfNotPresent",
+                "imagePullPolicy": "Always",
```

Comment:
@laurafitzgerald @matskiv It's more likely we will use a tagged version of the image. In this case, let's not always pull the latest image